src/client/sandbox/native-methods.js

```diff
@@ -175,15 +175,12 @@ class NativeMethods {
 
         this.DOMParserParseFromString = win.DOMParser.prototype.parseFromString;
 
-        // Setters
-        const inputValueDescriptor    = win.Object.getOwnPropertyDescriptor(win.HTMLInputElement.prototype, 'value');
-        const textAreaValueDescriptor = win.Object.getOwnPropertyDescriptor(win.HTMLTextAreaElement.prototype, 'value');
-
-        if (inputValueDescriptor && typeof inputValueDescriptor.set === 'function')
-            this.inputValueSetter = inputValueDescriptor.set;
+        const objectDataDescriptor = win.Object.getOwnPropertyDescriptor(win.HTMLObjectElement.prototype, 'data');
 
-        if (textAreaValueDescriptor && typeof textAreaValueDescriptor.set === 'function')
-            this.textAreaValueSetter = textAreaValueDescriptor.set;
+        // Setters
+        this.inputValueSetter    = win.Object.getOwnPropertyDescriptor(win.HTMLInputElement.prototype, 'value').set;
+        this.textAreaValueSetter = win.Object.getOwnPropertyDescriptor(win.HTMLTextAreaElement.prototype, 'value').set;
+        this.objectDataSetter    = objectDataDescriptor.set;
```

Comment:
Why object data getter definition
`this.objectDataSetter    = objectDataDescriptor.set;`
placed not on the next line after
`this.objectDataGetter               = objectDataDescriptor.get;`
?

Reply:
Because functions grouped by setters and getters.

Reply:
Ok